build.gradle

```diff
@@ -17,13 +17,13 @@ plugins {
     id 'net.researchgate.release' version '2.6.0'
 }
 
-ext.opentracingVersion = getProperty('opentracingVersion','0.32.0')
+ext.opentracingVersion = getProperty('opentracingVersion','0.33.0')
 ext.guavaVersion = getProperty('guavaVersion','18.0')
 ext.apacheThriftVersion = getProperty('apacheThriftVersion','0.12.0')
 ext.jerseyVersion = getProperty('jerseyVersion','2.22.2')
 ext.slf4jVersion = getProperty('slf4jVersion','1.7.25')
 ext.gsonVersion = getProperty('gsonVersion','2.8.2')
-ext.tracerResolverVersion = getProperty('tracerResolverVersion','0.1.6')
+ext.tracerResolverVersion = getProperty('tracerResolverVersion','0.1.7')
```

Comment:
Version 0.1.8 has been released with a couple of changes (adding module name and supporting classloader parameter).